### PR TITLE
github release

### PR DESCRIPTION
**Pull request by maddygriz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,58 @@
...
+
+To learn how to start using SpatialOmicsOverlay, view documentation for the
+version of this package installed in your system, start R and enter:
+
+``` r
+browseVignettes("SpatialOmicsOverlay")
+```
+
+or download vignette from NanoString website
+<>
```
Comment (NicoleEO) [inline README.md:42]: Remove or put pending

File: tests/testthat/test_utils.R [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+})
+
+testthat::test_that("mouse brain tiff can be downloaded",{
+    #Spec 1. The function downloads the mouse brain tiff and returns a valid 
+    #           file path.
+    tifFile <- downloadMouseBrainImage()
+    
+    expect_true(endsWith(tifFile, "mu_brain_004.ome.tiff"))
+    expect_true(file.exists(tifFile))
 })
```
Comment (NicoleEO) [inline tests/testthat/test_utils.R:55]: Add line

**Review by NicoleEO — CHANGES_REQUESTED** [review]
Small edits. Also, did you mean for the html version of the vignette to be included? Just checking.

**Comment by maddygriz** [comment]
I did intend for the html to be there but it doesn't render like I was expecting so it is probably best to remove it

**Review by NicoleEO — APPROVED** [review]

**Pull request merged**